cpp/velox/shuffle/VeloxShuffleWriter.cc

```diff
@@ -343,6 +337,13 @@ std::shared_ptr<arrow::RecordBatch> makeUncompressedRecordBatch(
   }
   return arrow::RecordBatch::Make(writeSchema, 1, {arrays});
 }
+
+inline bool shrinkBeforeSpill(SplitState state) {
```

Comment:
the `inline` is unnecessary, the compiler will make the decision when she sees the function defined in the inner of the class.